mcs/build/tests.make

```diff
@@ -75,7 +75,7 @@ test_assembly_dep = $(the_assembly)
 endif
 tests_CLEAN_FILES += $(ASSEMBLY:$(ASSEMBLY_EXT)=_test*.dll) $(ASSEMBLY:$(ASSEMBLY_EXT)=_test*.pdb) $(test_response) $(test_makefrag)
 
-xtest_sourcefile = $(PROFILE)_$(ASSEMBLY:$(ASSEMBLY_EXT)=_xtest.dll.sources)
+xtest_sourcefile = $(PROFILE_PLATFORM)_$(PROFILE)_$(ASSEMBLY:$(ASSEMBLY_EXT)=_xtest.dll.sources)
```

Comment:
This is making xunit tests platform-specific. I assume this works because we don't have profile-specific xunit tests (e.g. for nunit we have corlib_test.dll.sources, monodroid_corlib_test.dll.sources, etc.), right?

Reply:
@akoeplinger exactly, I have a couple of windows-only Registry xunit tests